<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>markdown_editor</class>
 <widget class="QWidget" name="markdown_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="edit_title_layout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Title </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_title">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="edit_format_layout">
     <item>
      <widget class="QToolButton" name="edit_bold">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="edit_italic">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>I</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="edit_wikilink">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #9797ff;</string>
       </property>
       <property name="text">
        <string>[[Page]]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="edit_link">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #9797ff;</string>
       </property>
       <property name="text">
        <string>[Text](url)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="tool_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="edit_save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="SpellTextEdit" name="page_edit">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpellTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">tritki.gui.spelltextedit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
